rust/gui-client/src-common/src/deep_link/windows.rs

```diff
@@ -68,23 +69,24 @@ async fn bind_to_pipe(pipe_path: &str) -> Result<named_pipe::NamedPipeServer, su
     for i in 0..NUM_ITERS {
         match create_pipe_server(pipe_path) {
             Ok(server) => return Ok(server),
-            Err(super::Error::CantListen) => {
-                tracing::warn!("`create_pipe_server` failed, sleeping... (loop {i})");
+            Err(e) => {
+                tracing::warn!(
+                    error = std_dyn_err(&e),
+                    "`create_pipe_server` failed, sleeping... (attempt {i}/{NUM_ITERS})"
```

Comment:
This will probably have the same problem as #7199. We should move out `i` and `NUM_ITERS` in a future PR

Reply:
https://github.com/firezone/firezone/issues/7238